libs/common-system/pom.xml

```diff
@@ -38,6 +38,10 @@
             <artifactId>slf4j-api</artifactId>
             <version>${org.slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.servicemix.bundles</groupId>
+            <artifactId>org.apache.servicemix.bundles.jsr305</artifactId>
```

Comment:
❗️ Need `<scope>provided</scope>`

Reply:
@coyotesqrl mentioned the scope is inherited from the parent's dependency manager

Reply:
Okay. I see where that is happening 👍  We only benefit from that on master, not release branches.